datalab/utils/commands/_chart_data.py

```diff
@@ -30,7 +30,9 @@
 from . import _utils
 
 
-@IPython.core.magic.register_cell_magic
+# Disable the magic here because another one with same name is available under
```

Comment:
I want to keep it here to so the changes for datalab.bigquery to work independently are minimal.